src/consumer/consumer.js

```diff
@@ -159,8 +159,10 @@ export default class Consumer {
     const components = ids.map(async (id: BitId) => {
       let dependenciesTree = {};
       const dependencies = [];
+      const resolvedBitStringWithVersion = getLatestVersionNumber(bitMap.getAllComponents(), id.toString());
```

Comment:
Change the names to something like:
idWithConcreteVersionString and
idWithConcreteVersion